samples/helloworld.c

```diff
@@ -0,0 +1,8 @@
+// Traditional "Hello World" program
+
+#include <stdio.h>
+
+int main() {
+   printf("Hello, World!");
```

Comment:
The "traditional" hello world program includes a line feed - see e.g. https://en.wikipedia.org/wiki/%22Hello,_World!%22_program

Reply:
Very well.  Change made.